File: Java/VSCode/src/CelciusAFahrenheit.java

import java.util.Scanner;

public class CelciusAFahrenheit {
    
    static double fahr(double c){
        double temp = 1.8 *c + 32;
        return temp;
    }

    public static void main(String[] args) {
        try(Scanner sc = new Scanner(System.in)){
            System.out.println("Ingresa grados celcius:");
            double c = sc.nextDouble();
            System.out.println("Los grados Fahrenheit son: ");
            System.out.println(fahr(c));
        sc.close();
        }
    }
}// End of class CelciusAFahrenheit

File: Python/CelciusAFahrenheit.py
def fahr(c):
    temp = 1.8 * c + 32
    return temp

def main():
    c = float(input("Ingresa grados celsius: "))
    print("Los grados Fahrenheit son:")
    print(fahr(c))

if __name__ == "__main__":
    main()